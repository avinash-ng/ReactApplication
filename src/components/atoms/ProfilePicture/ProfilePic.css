.image-container {
    margin-left: 500px;
    width: 350px;
    height: 350px;
}

.image-container img {
    width: 300px;
    height: 300px;
}