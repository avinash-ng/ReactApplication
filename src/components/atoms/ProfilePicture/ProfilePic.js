import React from 'react'
import './ProfilePic.css'
const ProfilePic = (props) => {
    return (
        <div className="image-container">
            <img src={props.imageName} alt={props.altText} />
        </div>
    )
}

export default ProfilePic