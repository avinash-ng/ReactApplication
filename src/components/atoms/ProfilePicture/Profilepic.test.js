import ProfilePic from './ProfilePic'
import expect from 'expect'
import sinon from 'sinon';
import { shallow, configure } from 'enzyme'
import React from 'react'
import Adapter from 'enzyme-adapter-react-16';
// import renderer from 'react-test-renderer'
import { render } from '@testing-library/react';

configure({ adapter: new Adapter() });
describe('<ProfilePic /> ', () => {
    it('renders ProfilePic element', () => {
        const RenderComponent = shallow(
            <ProfilePic ></ProfilePic>
        );
        // console.log(RenderComponent)
        expect(RenderComponent.exists()).toBe(true)
    })

    it('recieved props correctly', () => {
        const alt = "Error in loading..."
        const srcImage = "photo.png"
        const RenderComponent = shallow(<ProfilePic imageName={srcImage} altText={alt}></ProfilePic>)
        expect(RenderComponent.find('img').prop('alt')).toEqual(alt)

    })


    it('Matches snapshot', () => {
       
            const RenderComponent = shallow(<ProfilePic></ProfilePic>)
            expect(RenderComponent).toMatchSnapshot()
    
    })
})