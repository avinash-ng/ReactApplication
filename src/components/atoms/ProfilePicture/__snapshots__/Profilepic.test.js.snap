// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfilePic />  Matches snapshot 1`] = `ShallowWrapper {}`;
